README.md

```diff
@@ -45,10 +86,11 @@ For bug reports, please [open an issue on GitHub](https://github.com/rollbar/rol
 
 To set up a development environment, you'll need Node.js and npm.
```

Comment:
This section has some duplicative information with CONTRIBUTING.md. Maybe this section should be renamed "Developing and Contributing" and replaced with a link to CONTRIBUTING.md?